controller/main.go

```diff
@@ -17,9 +17,11 @@ package main
 import (
 	"flag"
 	"fmt"
+	"github.com/go-kit/kit/log/level"
```

Comment:
move it with github.com/go-kit/kit/log

Reply:
> Some nitpicks
> After those changes LGTM

sure, will take care of those small changes soon. Just one thing: I didn't get what you mean in your comments by
> move it with github.com/go-kit/kit/log

Can you explain?
Thanks.

Reply:
Put the 2 gokit imports one after the other